app/components/works/add_files_component.html.erb

```diff
@@ -75,7 +75,7 @@
       data: { action: 'complex-radio#disableUnselectedInputs file-uploads#updatePanelVisibility deposit-button#updateDepositButtonStatus'}
     %>
     <%= form.label :upload_type, "Upload one or more files (>10GB) via Globus web-based transfer software.", class: 'form-check-label fw-semibold' %>
-    <p><%= link_to "Use single sign-on to activate your account first if you don't already have one" , 'https://globus.stanford.edu/accounts/create.html', target: :blank%>, and then return here to complete this form.</p>
+    <p>Follow <%= link_to "these instructions", 'https://docs.google.com/document/d/10b7y3yZCOfyVJ_uP4l7QHbkILQKnfdJi9hj-MpmmdIk/edit', target: :blank %> to set up your Globus account and the necessary software, then return here to complete your deposit.</p>
```

Comment:
I think you can use the `Settings.help_doc_url` here instead of hard coding it. It's used in a couple of the emails too.